server/services/assessRisksAndNeedsService.test.ts

```diff
@@ -40,14 +40,20 @@ describe(AssessRisksAndNeedsService, () => {
     })
 
     it('provides a userMessage on failure', async () => {
-      restClientMock.get.mockRejectedValue(createError(404))
+      restClientMock.get.mockRejectedValue(createError(409))
```

Comment:
is it worth adding a test for a 5xx branch as well?

Reply:
done